diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -132,7 +132,8 @@ InterfaceConfiguration DiffDriveController::state_interface_configuration() cons
   return {interface_configuration_type::INDIVIDUAL, conf_names};
 }
 
-controller_interface::return_type DiffDriveController::update()
+controller_interface::return_type DiffDriveController::update(
+  const rclcpp::Time & time, const rclcpp::Duration & period)
```

Comment:
Should we remove the `period` variable name so it doesn't complain about unused variables?
```suggestion
  const rclcpp::Time & time, const rclcpp::Duration & /*period*/)
```